tiatoolbox/annotation/storage.py

```diff
@@ -2585,9 +2671,9 @@ def _handle_str_pquery(
 
     @staticmethod
     def _kind_of_pquery(
-        select: Union[str, bytes, Callable],
-        where: Union[str, bytes, Callable],
-    ) -> Tuple[bool, bool, bool]:
+        select: str | bytes | Callable,
+        where: str | bytes | Callable,
```

Comment:
Again we could use custom defined types here, Select and Predicate

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/991322d859c0c22e7c7a1d713362944aa4317198